apps/nully/docker/coturn/Dockerfile.sliplane

```
@@ -0,0 +1,204 @@
+
+# Create necessary directories
+RUN mkdir -p /var/log/coturn /etc/coturn/certs
+RUN chown -R nobody:nogroup /var/log/coturn
+
+# Create base configuration file inline
+RUN cat > /etc/coturn/turnserver.conf << 'EOF'
+# CoTURN Configuration for Nully App
+# High-performance TURN server configuration
+
+# Listening interfaces and ports
+listening-port=3478
```

Hadolint (2.12.0)

[error] unexpected 'l'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)